src/content/docs/developer/modules/other/external-qa-checks.mdx

```diff
@@ -0,0 +1,246 @@
+
+## Access
+
+You can grant access to this module to one of the following user categories:
+
+For Crowdin:
+
+* Only me (i.e., project owner)
+* All project members
+* Selected users
+
+For Crowdin Enterprise:
+
+* Only organization admins
+* All users in the organization projects
+* Selected users
+
+## Structure
+
+```json title="manifest.json"
+{
+  "modules": {
+    "external-qa-check": [
+      {
+        "key": "custom-check-qa",
+        "name": "QA Check",
+        "description": "Description",
+        "runQaCheckUrl": "/validate",
+        "getBatchSizeUrl": "/batch-size",
+        "url": "/settings/index.html"
+      }
+    ]
+  }
+}
+```
+
+## Properties
+
+<table>
+  <tbody>
+    <tr>
+      <td><code>key</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> Module identifier within the Crowdin app.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>name</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> The human-readable name of the module.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>description</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Description:</strong> The human-readable description of what the module does.<br/>
+         The description will be visible in the Crowdin Enterprise UI.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>runQaCheckUrl</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> The relative URL triggered when sending texts for QA validation.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code class="whitespace-nowrap">getBatchSizeUrl</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> The relative URL triggered when retrieving the batch size supported by the module.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>url</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> The relative URL to the module settings page.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>environments</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Allowed values:</strong> <code>crowdin-enterprise</code></p>
+        <p><strong>Description:</strong> Set of environments where a module could be installed.<br/>
+        This parameter is needed for cross-product applications.</p>
+      </td>
+    </tr>
```

Comment:
This field is missing in the structure example and has the wrong allowed values (only the `crowdin-enterprise`)

Reply:
> This field is missing in the structure example

For historical reasons, this is common to most articles about modules. Should I add the `environments` to the manifest.json structure in other articles as well?

> has the wrong allowed values (only the `crowdin-enterprise`)

This module is only available for Enterprise so the values are correct. Instead I removed the Crowdin-related info from the ## Access section.